src/Compilers/CSharp/Portable/Binder/SwitchExpressionBinder.cs

```diff
@@ -98,9 +98,13 @@ internal override BoundExpression BindSwitchExpressionCore(SwitchExpressionSynta
             {
                 if (n is BoundLeafDecisionDagNode leaf && leaf.Label == defaultLabel)
                 {
+                    bool unnamedEnumValue = false;
                     var samplePattern = PatternExplainer.SamplePatternForPathToDagNode(
-                        BoundDagTemp.ForOriginalInput(boundInputExpression), nodes, n, nullPaths: false, out bool requiresFalseWhenClause);
-                    ErrorCode warningCode = requiresFalseWhenClause ? ErrorCode.WRN_SwitchExpressionNotExhaustiveWithWhen : ErrorCode.WRN_SwitchExpressionNotExhaustive;
+                        BoundDagTemp.ForOriginalInput(boundInputExpression), nodes, n, nullPaths: false, out bool requiresFalseWhenClause, ref unnamedEnumValue);
```

Comment:
>ref [](start = 139, length = 3)

nit: why use `ref` rather than `out`?

Reply:
It does feel like this specific parameter could be made 'out' while the rest of the stuff inside remains as 'ref'--just have to assign a 'false' default value before passing the out parameter on to 'SamplePatternForTemp'.